room.h

```diff
@@ -90,6 +90,7 @@ namespace QMatrixClient
             QList<User*> membersLeft() const;
 
             Q_INVOKABLE QList<User*> users() const;
+			Q_INVOKABLE QStringList usernames() const;
```

Comment:
Can you please rewrite this as a `Q_PROPERTY` instead of `Q_INVOKABLE` function?

Also, consider using a `QList<User*>` in QML instead (basically, the output from `users()`). I understand it may be not so convenient, so I don't insist on this part.